adr/0015-activity-logging.md

```diff
@@ -0,0 +1,54 @@
+# 15. Activity Logging
+
+Date: 2024-10-02
+
+## Status
+
+Accepted
+
+## Context
+
+Trento feature-set is growing and not limited to only reading operations but expanding to applying state changes of different natures:
+- tagging/untagging resources
+- applying trento settings (api key, SUMA, etc...)
+- user/profile management operations (creation, modification, deletion)
+- operate on the monitred infrastructure (apply patches/upgrade packages via SUMA integration, apply remediations for failing checks)
+
+Such activities may be of huge impact and visibility on them is a great asset to troubleshooting incidents, determine root causes of issues and it improves transparency of the system in general.
```

Comment:
I guess `determining root causes ofi issues and improving transparencey` to follow with the same time term